bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/PJLinkDeviceHandler.java

```diff
@@ -129,8 +141,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
     logger.trace("Received command {} on channel {}", command, channelUID.getId());
     try {
       PJLinkDevice device = getDevice();
-      switch (channelUID.getId()) {
-        case CHANNEL_POWER:
+      @Nullable
```

Comment:
Put the @Nullable on the same line as the variable delcaration

Reply:
I would, but the ESH formatter configuration insists on moving it back to an extra line when I let it run. So I guess it's better to keep it as it is to avoid manual rework after each formatting run.

Reply:
On the other hand, I'm not even sure if it is needed at all, I'll check that again...

Reply:
This looks like the last open point. Can you have a look?

Reply:
I'll only have access to my dev environment at the end of next week. I've had a quick look before and I found that this @nullable is not required at all. I'll remove it as soon as I can access my dev env again.